test/JDBC/expected/BABEL-3802-vu-prepare.out

```diff
@@ -63,7 +67,10 @@ CREATE PROCEDURE BABEL_3802_vu_prepare_t_p3 AS (
         power(CAST(NULL AS SMALLINT),1),
         power(CAST(8969.32 AS SMALLINT),1),
         power(CAST(8962 AS SMALLINT),1.1),
-        power(CAST(8962 AS SMALLINT),1.2)
+        power(CAST(8962 AS SMALLINT),1.2),
+        power(CAST(NULL AS SMALLINT),NULL),
```

Comment:
Can you pls check for this test case as well :-
`Select power(cast(2 as smallint),-1)`
And pls add some testcase with negative power. Thanks

Reply:
Addressed